designs/PackageVulnerability/DotnetListPackageVulnerable.md

```diff
@@ -165,42 +140,43 @@ The following sources were used:
 
 Package                Requested   Resolved    Latest
 EntityFramework        6.1.2       6.1.2       6.2.0
-NUnit                  2.4.0 (D)   2.6.4       3.8.1  
-My.Sample.Pkg          2.1.3 (V)   4.1.0       4.1.0
+NUnit                  2.4.0       2.6.4 (D)   3.8.1  
+My.Sample.Pkg          2.1.3       4.1.0 (V)   4.1.0
 
+ > To see transitive packages, add the `--include-transitive` option.
```

Comment:
Just curious, why does this line start with a `>`. Does the `>` next to the markers look weird to anyone else? Can/should we be consistent and add a `>` to the markers' lines as well?

Reply:
Per @JonDouglas's comment above, we'll remove footers for now and let customers use `--help` for the full help experience.